docs/source/conf.py

```diff
@@ -24,10 +24,8 @@
 author = 'The Foreman'
 
 try:
-    import pkg_resources
-    package = pkg_resources.require("obal")[0]
-    version = package.version
-    release = package.version
+    from importlib import metadata
+    version = release = metadata.version('obal')
 except:
```

Comment:
Shall we scope it explicitly to `metadata.PackageNotFoundError` now?

I'd be tempted to use
```python
from importlib import metadata
try:
    version = release = metadata.version('obal')
except metadata.PackageNotFoundError:
    version = release = ''